src/app/pages/checkout-payment/formly/server-validation.extension.ts

```diff
@@ -0,0 +1,39 @@
+import { FormControl } from '@angular/forms';
+import { FormlyExtension, FormlyFieldConfig } from '@ngx-formly/core';
+
+export const serverValidationExtension: FormlyExtension = {
+  prePopulate: field => {
```

Comment:
Is this extension really special for the checkout payment page ? If not I would expect this extension under formly extensions